gui/wxpython/datacatalog/tree.py

```diff
@@ -1008,16 +1008,24 @@ def OnEndDrag(self, node, event):
         self.copy_mode = wx.GetMouseState().ControlDown()
         if node:
             self.DefineItems([node])
-            if self._restricted and gisenv()['MAPSET'] != self.selected_mapset[0].data['name']:
-                GMessage(_("To move or copy maps to other mapsets, unlock editing of other mapsets"),
+            if None not in self.selected_mapset:
+                if self._restricted and gisenv()['MAPSET'] != self.selected_mapset[0].data['name']:
+                    GMessage(_("To move or copy maps to other mapsets, unlock editing of other mapsets"),
+                             parent=self)
+                    event.Veto()
+                    return
+
+                event.Allow()
+                Debug.msg(1, "DROP DONE")
+                self.OnPasteMap(event)
+            else:
+                GMessage(_("To move or copy maps to other location, "
+                           "please drag them to mapset node, not "
+                           "location node"),
```

Comment:
> To move or copy maps to other location, please drag them to mapset node, not location node

I believe the concept of "node" (in this context) is very much a programmers vocabulary, maybe something like:

"To move or copy maps to other location, please drag them to a mapset in the destination location"

makes more sense by a regular user.

Reply:
Thanks for your suggestion. I've already fixed it.